main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
-    pass
+
+    with open("players.json") as f:
+        data = json.load(f)
+
+    for name in data:
+
+        # check if race exists
+        if Race.objects.filter(name=data[name]["race"]["name"]).exists():
+            race = Race.objects.get(name=data[name]["race"]["name"]).id
+        else:
+            Race.objects.create(name=data[name]["race"]["name"],
+                                description=data[name]["race"]["description"])
```

Comment:
You can use very comfortable method get_or_create
So you can merge get and create